module/VuFind/src/VuFind/Search/EDS/AbstractEDSParams.php

```diff
@@ -48,6 +48,14 @@
  */
 class AbstractEDSParams extends \VuFind\Search\Base\Params
 {
+    /**
+     * Fields that the EDS API will always filter multiple values using OR, not AND.
+     *
+     * @var array
+     */
+    protected $FORCED_OR_FIELDS = [
```

Comment:
Is there a compelling reason to name this property in ALL_CAPS_SNAKE_CASE? That's usually reserved for constants.

Reply:
Yeah I meant it as a "constant", in the sense of one that will be defined more specifically in the EDS-specific subclass, and potentially later in the EPF one as well, but should exist for either based on the mysterious logic behind the scenes of the APIs.  See my comment below about EDS/Params. I defined it in the abstract class so it can be used by the common `parseOperatorAndFieldName`.  If it's cleaner, it could be an abstract function `getForcedOrFields` instead.

Reply:
I don't have a strong feeling about whether it should be a property or a method -- just that if it's a property, I think it should be `$forcedOrFields` instead of `$FORCED_OR_FIELDS` to reflect that it's not really a constant. (I think you could also define it as `const FORCED_OR_FIELDS` but then you would of course be unable to change it. :-)

Reply:
Fair enough, I changed the capitalization since it's not technically a constant.  The user has no real control over it since it's up to the API, but there might be reasons to change it anyway.  Especially if we are misunderstanding the API in some way and find that out later, which in the case of EPF at least is quite likely.